types/model.d.ts

```diff
@@ -0,0 +1,37 @@
+import * as THREE from 'three'
+
+type Material = THREE.Material | THREE.Material[]
```

Comment:
Why can `Material` be `THREE.Material[]`?

Reply:
It's according to type definitions in three.js. If this type isn't an array of `Material`, I would remove it.

Reply:
You even don't need to define such a type. You can use `THREE.Material` directly.